src/modules/fancyzones/lib/Zone.h

```diff
@@ -1,13 +1,49 @@
 #pragma once
 
+/**
+ * Class representing one zone inside applied zone layout, which is basically wrapper around rectangle structure.
+ */
 interface __declspec(uuid("{8228E934-B6EF-402A-9892-15A1441BF8B0}")) IZone : public IUnknown
 {
+    /**
+     * @returns Zone coordinates (top-left and bottom-right corner) represented as RECT structure.
+     */
     IFACEMETHOD_(RECT, GetZoneRect)() = 0;
+    /**
+     * @returns Boolean indicating if zone is empty or there are windows assigned to it.
+     */
     IFACEMETHOD_(bool, IsEmpty)() = 0;
+    /**
+     * @param   window Window handle.
+     * @returns Boolean indicating of specified window is assigned to the zone.
+     */
     IFACEMETHOD_(bool, ContainsWindow)(HWND window) = 0;
+    /**
+     * Assign single window to this zone.
+     *
+     * @param   window     Handle of window which should be assigned to zone.
+     * @param   zoneWindow Handle of hidden window representing entire display work area.
```

Comment:
`Handle of hidden window` -> `The m_window of a ZoneWindow, a hidden window` representing entire display work area.